From e8a42493932ded3f65b80a3104d01838ae3505c4 Mon Sep 17 00:00:00 2001
From: Alessandro Astone <ales.astone@gmail.com>
Date: Wed, 20 Mar 2019 21:19:29 +0100
Subject: [PATCH 10/20] common: recovery: allow setting time

Fixes:
 avc:  denied  { sys_time } for  pid=567 comm="recovery"
 capability=25 scontext=u:r:recovery:s0 tcontext=u:r:recovery:s0
 tclass=capability permissive=0

Change-Id: I3a5b2f5cc749fbee56c17765042b9dc616d4fedd
---
 common/private/recovery.te | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/common/private/recovery.te b/common/private/recovery.te
index ac14b8f..ed81acd 100644
--- a/common/private/recovery.te
+++ b/common/private/recovery.te
@@ -31,4 +31,7 @@ domain_trans(recovery, otapreopt_chroot_exec, backuptool)
 
 # Volume manager
 allow recovery self:netlink_kobject_uevent_socket create_socket_perms_no_ioctl;
+
+# Set time
+allow recovery self:capability sys_time;
 ')
-- 
2.17.1

